template.md

```diff
@@ -0,0 +1,321 @@
+
+- Example: [Blockquotes](#blockquote)
+
+To link to a Markdown file in the same repo, use [relative links](https://www.w3.org/TR/WD-html40-970917/htmlweb.html#h-5.1.2), including the ".md" at the end of the filename.
+
+- Example: [Readme file](readme.md)
+- Example: [Welcome to .NET](./docs/welcome.md)
+
+To link to a header in a Markdown file in the same repo, use relative linking + hashtag linking.
+
+- Example: [.NET Community](./docs/welcome.md#community)
+
+### External Links
+
+To link to an external file, use the full URL as the link.
+
+- Example: [GitHub](http://www.github.com)
+
+If a URL appears in a Markdown file, it will be transformed into a clickable link.
+
+- Example: http://www.github.com
+
+### Links to APIs
+
+Our build system has some extensions that are compatible with GFM and allow us to link to .NET Core APIs without having to use external links.  
+When linking to APIs, you can use its unique identifier (UID) that are auto-generated from the source code.
+
+You can use one of the following syntax:
+1. Markdown link: `[link_text](xref:UID)`
+2. Auto link: `<xref:UID>`
+3. Shorthand form: `@UID`
+
+For example, @System.String or [String class](xref:System.String). For more information about using this notation, see [Using cross reference](https://dotnet.github.io/docfx/tutorial/links_and_cross_references.html#using-cross-reference).
+
+> Right now, there is no easy way to find the UIDs. The best way to find the UID for an API is to search for it in this repo: https://github.com/docascode/coreapi. We're working on having a better system in the future.
+
+## Lists
+
+### Ordered lists
+
+1. This 
+1. Is
+1. An
+1. Ordered
+1. List  
+
+
+#### Ordered list with an embedded list
+
+1. Here
+1. comes
+1. an
+1. embedded
+    1. Miss Scarlett
+    1. Professor Plum
+1. ordered
+1. list
+
+
+### Unordered Lists
+
+- This
+- is
+- a
+- bulleted
+- list
+
+
+##### Unordered list with an embedded list
+
+- This 
+- bulleted 
+- list
+    - Mrs. Peacock
+    - Mr. Green
+- contains  
+- other
+    1. Colonel Mustard
+    1. Mrs. White
+- lists
+
+
+## Horizontal rule
+
+---
+
+## Tables
+
+| Tables        | Are           | Cool  |
+| ------------- |:-------------:| -----:|
+| col 3 is      | right-aligned | $1600 |
+| col 2 is      | centered      |   $12 |
+| col 1 is default | left-aligned     |    $1 |
+
+
+## Code
+
+### Generic code block
+
+Use three backticks (&#96;&#96;&#96;) for generic code block coding. 
+
+```
+function fancyAlert(arg) {
+    if(arg) {
+        $.docs({div:'#foo'})
+    }
+}
+```
+
+### Code blocks with language identifier
+
+Use three backticks (&#96;&#96;&#96;) + a language ID to apply language-specific color coding to a code block. Here is the entire list of [GFM language IDs](https://github.com/jmm/gfm-lang-ids/wiki/GitHub-Flavored-Markdown-(GFM)-language-IDs).
+
+##### C&#9839;
```

Comment:
are you referring to your previous comment about just linking to markdown syntax and not cover this here @BillWagner?